src/target/riscv/riscv.c

```diff
@@ -4409,10 +4397,8 @@ COMMAND_HANDLER(riscv_reset_delays)
 
 COMMAND_HANDLER(riscv_set_ir)
 {
-	if (CMD_ARGC != 2) {
-		LOG_ERROR("Command takes exactly 2 arguments");
+	if (CMD_ARGC != 2)
```

Comment:
Please, fix the usage for this command (the first argument is not optional):
https://github.com/riscv-collab/riscv-openocd/blob/7f054ce0cf1a1dff5b69ce4b5965331d04a8673b/src/target/riscv/riscv.c#L5288